packer/centos65-hvm-64-base.json

```diff
@@ -0,0 +1,52 @@
+{
+  "builders": [
+    {
+        "type" : "amazon-ebssurrogate",
+        "region" : "us-east-1",
+        "ssh_username" : "ec2-user",
+        "instance_type" : "t2.medium",
+        "source_ami" : "ami-e9a18d80",
+        "ami_regions": [
+            "us-west-2"
+        ],
+        "ami_name": "centos65-hvm-64-base-{{timestamp}}",
+        "ami_description": "centos65-hvm-64-base EBS AMI",
+        "ami_virtualization_type": "hvm",
+        "subnet_id": "subnet-0aa98361",
+        "launch_block_device_mappings" : [
+            {
+            "volume_type" : "gp2",
+            "device_name" : "/dev/xvdf",
+            "delete_on_termination" : false,
+            "volume_size" : 10
+            }
+        ],
+        "ami_root_device": {
+            "source_device_name": "/dev/xvdf",
+            "device_name": "/dev/xvda",
+            "delete_on_termination": true,
+            "volume_size": 16,
```

Comment:
I completely forgot, do we need any resizing on boot or it's done by cloudinit?

Reply:
I'm not entirely sure how the partition was previously expanded with the aws sciprts but in this case, when an instance is provisioned, the partition will need to be adjusted using growpart.  At the end of the provisioning, the instance is rebooted and cloud-init resizes the filesystem on startup.  I've been working on a terraform example that includes a short provisioning script to expand the partition when the instance is being created.